pagefaultgames/pokerogue
# [Localization] Japanese locales

## What are the changes?

· Imported the Japanese translation from #726 ~~and the pokemon_emerald_pro.ttf file. pokems.ttf is not included.~~
· New japanese and kanji font "Galmuri11", "Galmuri9" from https://galmuri.quiple.dev/



## Why am I doing these changes?

· Addressed the issue as #726 was closed.
· I hope this will attract more Japanese users and bring in more translators. The descriptions for moves and abilities are too many for me to handle alone.

## What did change?

· Cleaned up line breaks and spaces.
· Translated phrases on the sign-up and login screens.
· Translated some messages that appear when a Pokémon evolves.
· Translated Item name and description (with berry)
· Translated ability trigger message
· Translated game mod name
· Translated save slot message
· Translated challenges option
· Translated tutorial text
· Since the descriptions for moves and abilities are still in English, updated those based on changes in the English version (e.g. Acrobatics).

· added that adjusts the font size to fit the UI when Japanese is selected. (ja only) https://github.com/Chepyu/pokerogue/pull/4
· Japanese font does not affect other language settings (especially Chinese characters) https://github.com/Chepyu/pokerogue/pull/5
· The font sizes have increased significantly, resulting in substantial optimization of the file size. https://github.com/Chepyu/pokerogue/pull/6

### Screenshots/Videos
![image](https://github.com/pagefaultgames/pokerogue/assets/33060456/805145d6-8c2b-491f-9437-91a25de6844e)
![image](https://github.com/pagefaultgames/pokerogue/assets/33060456/2cdaebf6-e4bd-40e0-87ab-ff040f2b2ff9)

## How to test the changes?
· Ran 'npm run test'
· Played classic mode on localhost
· Prepared data files with all Pokémon to check the UI on the starter screen

## Checklist
